dom-event-delegation/main.js

```diff
@@ -0,0 +1,13 @@
+var taskList = document.querySelectorAll('.task-list');
```

Comment:
while this does work, here you are only trying to select a single element, the element with the class of tasklist, so `querySelector` would be more appropriate. Note that you would not be required to use the for of loop below.